_pages/best-practices.html

```diff
@@ -0,0 +1,44 @@
+---
+layout: page
+permalink: best-practices/
+---
+
+<h1>Best practices</h1>
+<h2>Realtime updates</h2>
+
+<p>There is need to display actual information to client on buy, sell or expiration of contract. </p>
+
+<p>It is possible to use API-methods like
+    <a href="/api#portfolio"><code>portfolio</code></a>,
+    <a href="/api#profit_table"><code>profit_table</code></a>,
+    <a href="/api#statement"><code>statement</code></a>,
+    <a href="/api#balance"><code>balance</code></a>,
+but all they are "poll-model", i.e. slow and impose additional load on <a href="https://binary.com">Binary.com</a>,
+and as the cummulative effect might lead to lags or errors. Those calls are designed to be invoked at
```

Comment:
Change "Those" to "These", as you are still referring to the API calls mentioned earlier.